src/tiledb/cloud/_common/utils.py

```diff
@@ -112,3 +113,10 @@ def release_connection(resp: urllib3.HTTPResponse) -> None:
 
 def datetime_to_msec(t: Union[datetime.datetime, int, None]) -> Optional[int]:
     return int(t.timestamp() * 1000) if isinstance(t, datetime.datetime) else t
+
+
+def sanitize_enum_argument(enum_class: Type[Enum], value: str) -> Type[Enum]:
+    try:
+        return enum_class(value)
+    except ValueError:
+        raise ValueError(f"{value} is not a valid {enum_class.__name__}")
```

Comment:
This should be either a `raise ValueError(...) from None` or `raise ValueError(...) from ve` so that the error you’re raising here is treated as the “real” error and not a "while error A was being handled, error B happened" error